spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/utils/SparkUtils.scala

```diff
@@ -28,6 +28,25 @@ import org.apache.spark.sql.execution.QueryExecution
 import org.apache.spark.sql.hive.thriftserver.HiveThriftServer2
 
 object SparkUtils extends Logging {
+  private var _hiveConf: Configuration = _
+
+  private[atlas] def resetHiveConf: Unit = synchronized {
```

Comment:
This trick is needed to reflect changes on hive configuration. It should be only used for test.